packages/devtools_app/lib/src/screens/memory/panes/diff/controller/Item_controller.dart

```diff
@@ -59,7 +64,17 @@ class SnapshotListItem extends DiffListItem with AutoDisposeControllerMixin {
 
   var sorting = ColumnSorting();
 
-  final diffWith = ValueNotifier<SnapshotListItem?>(null);
+  final _diffWith = ValueNotifier<SnapshotListItem?>(null);
+  ValueListenable<SnapshotListItem?> get diffWith => _diffWith;
+  void setDiffWith(SnapshotListItem? value) {
```

Comment:
same comment around ordering. Also, any reason we aren't using a formal setter here?

Reply:
fixed ordering
formal setter wants to have the same type as the formal getter with the same name